### DISCUSSION TITLE
How to use the auto updater with electron-vite

### DISCUSSION THREAD

**Discussion by rayanarfawi** [alex8088/electron-vite#124] [Q&A]
is there any snippets on how to use the app update ? 
and how to check for the update on the main ?

**Comment by otekdo** [comment]
Same situation, would like to know the most recommended `way` if electron-vite handles app updates with this framework.
Is it possible to set up OTA updates

**Comment by otekdo** [comment]
@alex8088 Can https://www.electronjs.org/docs/latest/api/auto-updater work out of the box with `electron-vite`?

**Comment by otekdo** [comment]
@alex8088  I tried to have OTA updates with github Releases, the following way: 
https://www.electronjs.org/docs/latest/tutorial/updates#using-other-update-services
Sadly, that way did not work

**Reply by otekdo** [reply]
I used Hazel as the update service

**Comment by hrithikdas88** [comment]
Any updates on how to implement auto-update in this template
